### PR TITLE
added Biggest_of_n_numbers.java

### PR DESCRIPTION
**Pull request by Nidakhan786**
**Fixes issue:**
N/A

**Changes:**
Added Biggest_of_n_numbers.java

### CODE REVIEW

File: code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import java.util.Scanner;
+
+public class Biggest_of_n_numbers
+  {
+  public static void main(String argn[]){
```
Comment (arnavb) [inline code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java]: Horrible, horrible indentation! Fix it please! Look at the [style guide](https://github.com/OpenGenus/cosmos/blob/master/guides/coding_style/java/README.md#whitespace)!

File: code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    Scanner data = new Scanner(System.in);
+    int n, i, temp, max;
+    System.out.print("Enter numbers of elements");
+    n = data.nextInt();
```
Comment (arnavb) [inline code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java]: `int n =`

File: code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    System.out.println("Enter " +n+ " numbers");
+    max = data.nextInt();
```
Comment (arnavb) [inline code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java]: `int max = `

File: code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    for (i = 1; i < n; ++i)
```
Comment (arnavb) [inline code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java]: `for (int i = `

File: code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  public static void main(String argn[]){
+    Scanner data = new Scanner(System.in);
+    int n, i, temp, max;
+    System.out.print("Enter numbers of elements");
+    n = data.nextInt();
+    System.out.println("Enter " +n+ " numbers");
+    max = data.nextInt();
+    for (i = 1; i < n; ++i) 
+{ 
+    temp = data.nextInt();
```
Comment (arnavb) [inline code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java]: `int temp = `

File: code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java [hunk 6]
```diff
@@ -0,0 +1,22 @@
+import java.util.Scanner;
+
+public class Biggest_of_n_numbers
```
Comment (arnavb) [inline code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java]: Should be `BiggestOfNNumbers`

File: code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java (hunk 1 above)
Comment (arnavb) [inline code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java]: Usually it's called `String[] args`.

File: code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+            System.out.print("Enter numbers of elements");
+            int n = data.nextInt();
+            System.out.println("Enter " +n+ " numbers");
+            int max = data.nextInt();
+         
+            for (int i = 1; i < n; ++i)
+            { 
+                  int temp = data.nextInt();
+                  if(temp < max)
+                  continue;
```
Comment (arnavb) [inline code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java]: Fix indentaion here as well.

File: code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java (hunk 1 above)
Comment (Nidakhan786) [reply code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java]: okay

File: code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+                  else
+                  max = temp;
```
Comment (arnavb) [inline code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java]: Here too.

File: code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java (hunk 2 above)
Comment (Nidakhan786) [reply code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java]: done

File: code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+            }
+              System.out.println("Maximum is "+max);
```
Comment (arnavb) [inline code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java]: Here too. Also add space after + sign.

File: code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java (hunk 3 above)
Comment (Nidakhan786) [reply code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java]: done

File: code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java (hunk 4 above)
Comment (Nidakhan786) [reply code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java]: done

File: code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java [hunk 10]
```diff
@@ -0,0 +1,23 @@
+import java.util.Scanner;
+
+public class BiggestOfNNumbers
+{
+       public static void main(String[] args)
+       {
+            Scanner data = new Scanner(System.in);
+            System.out.print("Enter numbers of elements");
+            int n = data.nextInt();
+            System.out.println("Enter " +n+ " numbers");
```
Comment (arnavb) [inline code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java]: Add spaces around + signs

File: code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java (hunk 5 above)
Comment (Nidakhan786) [reply code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java]: done

File: code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java (hunk 6 above)
Comment (Nidakhan786) [reply code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java]: done

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Comment by Nidakhan786** [comment]
@AdiChat  Thank You 😄

**Pull request merged**